io/plugins/eu.esdihumboldt.hale.io.haleconnect.ui/src/eu/esdihumboldt/hale/io/haleconnect/ui/projects/ShareProjectWizard.java

```diff
@@ -47,14 +45,14 @@
 
 	private static final ALogger log = ALoggerFactory.getLogger(ShareProjectWizard.class);
 
-	private final HaleConnectService haleConnect;
+//	private final HaleConnectService haleConnect;
```

Comment:
Just thought that it is there because it might be needed later, and adapting the constructor now just to revert it some point might not make sense. But if you think it's obsolete then surely it can be removed.